Apps/Playground/CMakeLists.txt

```diff
@@ -173,6 +174,8 @@ if(APPLE)
             FOLDER "Playground")
     endif()
     set_property(TARGET Playground PROPERTY UNITY_BUILD false)
+    set_property(TARGET Playground PROPERTY XCODE_GENERATE_SCHEME YES)
```

Comment:
This property `XCODE_GENERATE_SCHEME` removes unnecessary schemes from Xcode.

Now Xcode shows only necessary schemes as valid run options:

![CleanShot 2024-08-23 at 15 47 32@2x](https://github.com/user-attachments/assets/6130b46a-0345-4971-848e-9db7c59ae87a)